Dockerfile

```diff
@@ -59,8 +75,6 @@ USER user
 
 WORKDIR /home/user
 
-ARG PROJECT=ethereum-poa-relay
```

Comment:
This breaks shared base cache between different `PROJECTs`.

Reply:
even if I put it on [L50](https://github.com/paritytech/parity-bridges-common/pull/779/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R50)?

Reply:
Yeah, the point is that commands between `50-62` do not depend on `PROJECT` so they can be re-used. It's not much (like only `libssl-dev/curl` installation that might take some time), but it's always something (when you build locally).

Maybe just move all the `ARGs` and `LABEL` to this line instead? I know it's nice to have all parameters at the top, but it does affect docker caching ability.